generators/ci-cd/templates/jenkins/Jenkinsfile.ejs

```diff
@@ -184,10 +184,10 @@ node {
 <%= gitLabIndent %>        // A pre-requisite to this step is to setup authentication to the docker registry
     <%_ if (buildToolMaven) { _%>
 <%= gitLabIndent %>        // https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#authentication-methods
-<%= gitLabIndent %>        sh "./mvnw -ntp jib:build"
+<%= gitLabIndent %>        sh "./mvnw -ntp package -Pprod jib:build"
     <%_ } else if (buildToolGradle) { _%>
 <%= gitLabIndent %>        // https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#authentication-methods
-<%= gitLabIndent %>        sh "./gradlew jib"
+<%= gitLabIndent %>        sh "./gradlew bootJar jib -Pprod -PnodeInstall --no-daemon"
```

Comment:
```suggestion
<%= gitLabIndent %>        sh "./gradlew bootJar jib -Pprod --no-daemon"
```

nodeInstall should be removed for consistency with everything else.
Please add it commented to gradle.properties:
```

## Install and use a local version of node and npm for consistency when using different machines.
# nodeInstall
```

You can propose to use by default but it should be another PR.

Reply:
But it is used in (almost) all Gradle tasks here in Jenkins pipeline. See e.g. backed or frontend tests.

Reply:
Your are right.
This can lead to different behaviors in different situations, but since this used in other places it's ok.

Can you add the config to gradle.properties anyway?

Reply:
Thanks.

Reply:
I've added the section to the properties.

And yes, I already have encountered some problems with this, because the default npm version of the LTS node is 6.14 and the specified version in the Gradle build file is 7.8. ☹️

Reply:
That should be tracked by https://github.com/jhipster/generator-jhipster/issues/14249.